examples/src/main/scala/com/example/http4s/ExampleService.scala

```diff
@@ -1,19 +1,18 @@
 package com.example.http4s
 
-import _root_.io.circe.Json
 import cats._
 import cats.effect.IO
 import cats.implicits._
 import fs2._
-import org.http4s.MediaType._
+import _root_.io.circe.Json
```

Comment:
This is annoying.  If we import `fs2._` first, `io.circe` needs a root import.  But now it's no longer lexicographic.  And if we move it to to the top, it's no longer clear why it needs to be a _root_ import.

Reply:
Ugh.

Reply:
I just import circe as root always. io is common enough that is conflicts too often. Now we have an io package as well.